rllib/examples/self_play_with_open_spiel.py

```diff
@@ -201,7 +217,7 @@ def policy_mapping_fn(agent_id, episode, worker, **kwargs):
 if __name__ == "__main__":
 
     args = get_cli_args()
-    ray.init(num_cpus=args.num_cpus or None, include_dashboard=False)
+    ray.init(num_cpus=args.num_cpus or None, include_dashboard=False, local_mode=True)
```

Comment:
can you remove local mode over here?

Reply:
oh yeah. Thanks for the reminder. After these tests ....